### ISSUE TITLE
Allow admins to remove the "long overdue" message from requests

### ISSUE THREAD

**Issue by HelenWDTK** [mysociety/alaveteli#7959]
We've been dealing with a query from a public authority who objected to the long overdue message appearing on a request that they had answered in line with the law. 

They provided a s12 refusal within the deadline, which the user marked as refused, the user then submitted a narrowed clarification on the same thread, but some time later. The authority is processing this, which is why it was correctly marked as waiting response, but due to the delay, the system was showing it as being over due which is misleading.

It would be good to have some way to remove the long overdue message from requests in these circumstances. To solve the issue, I had to mark the request as refused, which doesn't best reflect the current state of it, so is not ideal.

**Comment by garethrees** [comment]
I believe this can be done by using the "Was clarification request" on the relevant incoming message (also available under the "Actions" heading when clicking through to the incoming message).

![Screenshot 2023-10-16 at 13 03 49](https://github.com/mysociety/alaveteli/assets/282788/e8734aa1-66f1-485b-97bb-e7b27dc3b743)

**Comment by mdeuk** [comment]
> I believe this can be done by using the "Was clarification request"

Yep, it can - however would this not have the side effect of having the previous response “written off”, so to speak, as a request for clarification?

it would be helpful if we had a way to do this more generically, e.g. perhaps a dropdown giving a few options for updating the timeline (`clarification`, `new request` etc)

**Comment by HelenWDTK** [comment]
> would this not have the side effect of having the previous response “written off”, so to speak, as a request for clarification?

Yep, and that's half the issue. It's not technically a clarification request, but was instead a valid refusal. I like the dropdown idea.
